profile_bluesky/startup/instrument/devices/monochromator.py

```diff
@@ -0,0 +1,26 @@
+"""
+Monochromator motors
+"""
+
+__all__ = [
+    'mono',
+    ]
+
+from ..session_logs import logger
+logger.info(__file__)
+
+from apstools.devices import KohzuSeqCtl_Monochromator
+from ophyd import Component, EpicsMotor, EpicsSignal
+
+class Monochromator(KohzuSeqCtl_Monochromator):
+
+    th = Component(EpicsMotor,'m1', labels=('motor','monochromator'))  # Kohzu Theta # home_slew_rate=0
```

Comment:
For a simulator, right?

Reply:
Why do you say that? The `apstools.devices.KohzuSeqCtl_Monochromator` has only the theta `EpicsSignal`. It'd be good to have y, z there just for the record. chi and thf are needed, we scan them often to align the beamline.

Reply:
Added an issue to apstools (https://github.com/BCDA-APS/apstools/issues/402).  I'll follow what you add here.

Reply:
[Added](https://github.com/BCDA-APS/apstools/pull/405) in apstools just now for next release.  Using `y1` and `z2` as attribute names and adding `move_energy(keV)` method for command-line use to move to a new energy.

Suggest changing names here to `y1` and `z2` as well so transition to next apstools release will be easy.

Reply:
Our mono has a `y2` rather than `y1` (it's the second crystal that moves). Once the next apstools is released, should we havethe following?
```
y1 = None
y2 = Component(EpicsSignalRO, "KohzuYRdbkAI")
```